bands/models.py

```diff
@@ -42,6 +43,9 @@ class Band(MagModel):
     estimated_loadin_minutes = Column(Integer, default=c.DEFAULT_LOADIN_MINUTES, admin_only=True)
     estimated_performance_minutes = Column(Integer, default=c.DEFAULT_PERFORMANCE_MINUTES, admin_only=True)
 
+    def deadline_from_model(self, model):
```

Comment:
This is how we check to see if a step is applicable to a group based on its type -- if a deadline exists, then we assume that the group must complete that step of the checklist. Otherwise, we leave it off.